scripts/speaker_tasks/scp_to_manifest.py

```diff
@@ -31,46 +30,74 @@
 This scipt converts a scp file where each line contains  
 <absolute path of wav file> 
 to a manifest json file. 
+Optionally post processes the manifest file to create dev and train split for speaker embedding 
+training, also optionally chunk an audio file in to segments of random DURATIONS and create those
+wav files in CWD. 
+
+While creating chunks, if audio is not sampled at 16Khz, it resamples to 16Khz and write the wav file.
 Args: 
 --scp: scp file name
```

Comment:
I think this is a good time for doing "scp-> filelist" refactor, although you need to change the name of the file.
Two problems: 
(1) What you are using is not a SCP file.
(2) Even if it is SCP file, SCP file format is very hard to find the description and there is no explanation about scp file format in NeMo. 

I believe this script is quite important to prepare dataset for training speaker embedding model. I think changing the name is necessary.

Reply:
Updated